src/components/signin/actions.js

```diff
@@ -16,25 +16,25 @@ export const signinRequest = payload => ({
  * Triggers request to update redux store on success
  *
  * @function
- * @param {Object} payload An object containing signin status and data 
- * returned from siginn endpoint
+ * @param {boolean} isSuccessful returns true if isSuccessful else false.
  * @return {Object} The {@link actionTypes.SIGNIN_SUCCESS SIGNIN_SUCCESS} action.
  */
-export const signinSuccess = payload => ({
-    payload,
+export const signinSuccess = isSuccessful => ({
```

Comment:
Are you ever using this action creator where you pass in `isSuccessful` as `false`? If not, then you don't need to pass in `isSuccessful` as a parameter. We can assume in the reducer that this action is only called on success signin.